readme.md

```diff
@@ -1,7 +1,7 @@
-# GoAxios - inspired by the popular JavaScript Axios
```

Comment:
This line should remain.... the idea was actually suggested from Golang's Reddit channel 😉